operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -0,0 +1,263 @@
+image that is installed with CSM.
+
+1. (`ncn-mw#`) Find the appropriate barebones image and record the id.
+
+    ```bash
+    cray ims images list | grep -B 5 compute-csm.*aarch64
+    ```
+
+    Expected output will resemble:
+
+    ```text
+    [[results]]
+    arch = "aarch64"
+    created = "2024-01-10T22:48:48.277640+00:00"
+    id = "f9de6a5b-b49d-4e7a-b78f-18599a8e61f9"
+    name = "compute-csm-1.5-5.2.47-aarch64"
+    ```
+
+2. (`ncn-mw#`) Create a CFS configuration to customize the barebones image resembling the following.
+
+    ```json
+    {
+        "last_updated": "2020-09-22T19:56:32Z",
+        "layers": [
+            {
+                "clone_url": "https://api-gw-service-nmn.local/vcs/cray/configmanagement.git",
+                "commit": "01b8083dd89c394675f3a6955914f344b90581e2",
+                "playbook": "ims-computes.yml"
+            }
+        ],
+        "name": "ims-remote-build-node"
+    }
+    ```
+
+3. (`ncn-mw#`) Use CFS to customize the barebones image.
+
+    After posting your CFS configuration you will use it to customize the image id retrieved in the step one.
+
+    ```bash
+    cray cfs sessions create --target-group Application <IMS ID of your image> --target-image-map <IMS ID of your image> <name you want your new image to have in IMS> --target-definition image --name <pick a name for your CFS session> --configuration-name <name of the CFS configuration you created in the previous step>
+    ```
+
+4. (`ncn-mw#`) Boot the compute node with the customized image
+
+    Once the `cfs` customization is finished, the image is ready to be booted. Create a BOS session template referencing that image and use it to boot an arm node.
+
+    ```json
+    {
+      "boot_sets": {
+        "compute": {
+          "kernel_parameters": "ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN} root=live:s3://boot-images/<image id>/rootfs nmd_data=url=s3://boot-images/<image id>/rootfs,etag=<etag>",
+          "node_roles_groups": [ "Compute"],
+          "etag": "<etag>",
+          "arch": "ARM",
+          "path": "s3://boot-images/<image id>/manifest.json",
+          "rootfs_provider": "",
+          "rootfs_provider_passthrough": "",
+          "type": "s3"
+          }
+      }
+    }
+    ```
+
+### Adding storage to a remote build node
+
+1. Add storage to the remote build node.
+
+    By default compute nodes have limited storage. While executing small image builds may be possible, you will not be able to build larger images without additional
+    storage being available to the IMS builder node. We can achieve this by mounting ceph storage into the IMS builder node.
+
+    Below are a set of commands that should provide the IMS builder node with the storage that it needs to execute larger image builds.
+
+    (`cn#`) Search for the static DNS list and servers and set the environment.
+
+    ```bash
+    egrep 'NETCONFIG_DNS_STATIC_SERVERS|NETCONFIG_DNS_STATIC_SEARCHLIST' /etc/sysconfig/network/config
```

Comment:
I'm not entirely clear on this step. Is the intention to set those two environment variables to be equal to the values that they have in the `/etc/sysconfig/network/config` file?

Reply:
Added steps for better clarity regarding this step